docs/_posts/2022-05-06-writing-cli-tools-with-typescript.md

```diff
@@ -0,0 +1,187 @@
+# Writing CLI tools with TypeScript
```

Comment:
Your post is lacking meta information, so your post will be ignored.
Also please add the exact time you wish for it to be "published". It won't get rendered before even though it's merged. 

```suggestion
---
layout: post
title:  Writing CLI tools with TypeScript
date:   2022-05-06 08:00:13 +0100
categories: typescript, cli
author: Friedrich Kurz
---
```

Reply:
I added some docs on how to run jekyll locally using docker only:
https://github.com/weeklyjs/weeklyjs.github.io#running-locally

Reply:
I added an excerpt attribute to the YAML frontmatter btw, so I can keep my headings in the main blog post.